packages/exporter-trace-otlp-http/src/OTLPExporterBase.ts

```diff
@@ -102,6 +110,26 @@ export abstract class OTLPExporterBase<
     });
   }
 
+  private _configureTimeout(timeoutMillis: number | undefined): number {
+    let defaultTimeout = getEnv().OTEL_EXPORTER_OTLP_TRACES_TIMEOUT;
+
+    if ( defaultTimeout === null) {
+      defaultTimeout = getEnv().OTEL_EXPORTER_OTLP_TIMEOUT;
+    }
+
+    return typeof timeoutMillis === 'number'
+        ? timeoutMillis < 0
+          ? this._invalidTimeout(timeoutMillis, defaultTimeout)
+          : timeoutMillis
+            : defaultTimeout;
+  }
+
+  private _invalidTimeout(timeout: number, defaultTimeout: number): number {
```

Comment:
i think you should inline this check inside the above config

Reply:
This is fixed.